packages/liferay-npm-scripts/src/utils/mergeBabelLoaderOptions.js

```diff
@@ -22,30 +22,45 @@ function mergeBabelLoaderOptions(webpackConfig) {
 		return webpackConfig;
 	}
 
-	webpackConfig.module.rules.forEach((rule) => {
-		let {use} = rule;
-
-		if (!use) {
-			return;
-		}
-
-		if (!Array.isArray(use)) {
-			use = [use];
-		}
-
-		use.forEach((useEntry, i) => {
-			if (typeof useEntry === 'string') {
-				use[i] = {
-					loader: useEntry,
-					options: {...BABEL_CONFIG},
+	return {
+		...webpackConfig,
+		module: {
+			...webpackConfig.module,
+			rules: webpackConfig.module.rules.map((rule) => {
+				const {use} = rule;
+
+				if (!use) {
+					return rule;
+				}
+
+				const mergeUseEntry = (useEntry) => {
+					if (
+						typeof useEntry === 'string' &&
+						useEntry === 'babel-loader'
```

Comment:
You don't need the `typeof` check because your `=== 'babel-loader'` already effectively includes such a check.